src/index.ts

```diff
@@ -23,6 +23,10 @@ const argv = yargs
     describe: 'The number of intermediate orderbooks to look at when computing the transitive one',
     default: 2,
   })
+  .option('pollFrequency', {
```

Comment:
ubernit: slightly inconsistent arg names - would prefer is the casing (camelCase or snake_case) were consistent for all the args.

Reply:
For cli arguments, usually kebap case is more common, right?

Reply:
From the yargs docs:
```
var argv = require('yargs')
  .env('MY_PROGRAM')
  .option('f', {
    alias: 'fruit-thing',
    default: 'apple'
  })
  .argv
console.log(argv)
```
```
$ node fruity.js
{ _: [],
  f: 'apple',
  'fruit-thing': 'apple',
  fruitThing: 'apple',
  '$0': 'fruity.js' }
```

So it looks like using kebab-case in the spec will give you camelCase properties. I wouldn't be surprised if using camelCase properties also allowed accepted arguments in kebab-case.

Reply:
There is an [open issue](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/39255) that the typescript type derivation doesn't work for the camelCase properties. But we can access the kebap case ones via `object[string]` (cf. latest commit)

Reply:
Today I learned about kebab-case!